tests/pagila/copydb.sh

```diff
@@ -61,6 +61,7 @@ pgcopydb clone --notice \
          --skip-ext-comments \
          --skip-db-properties \
          --estimate-table-sizes \
+         --split-max-parts=5 \
```

Comment:
Could we use the environment variable in the docker-compose file for this one? that would allow testing that parts of the code.

Reply:
I removed this line and added environment variable in docker-compose. I verified that we still limit the splits to 5 on my local test environment.